src/views/nft/components/CollectibleCard/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box, Card, CardBody, CardProps, Flex, Text } from '@pancakeswap/uikit'
+import { useBNBBusdPrice } from 'hooks/useBUSDPrice'
+import PreviewImage from './PreviewImage'
+import { CostLabel, MetaRow, ProfileNftTag, SellingNftTag, WalletNftTag } from './styles'
+import { Collectible } from './types'
+
+export interface CollectibleCardProps extends CardProps {
+  collectible: Collectible
+}
+
+const StyledCollectibleCard = styled(Card)`
+  border-radius: 8px;
+  transition: opacity 200ms;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 320px;
+
+  & > div {
+    border-radius: 8px;
+  }
+`
+
+const CollectibleCard: React.FC<CollectibleCardProps> = ({ collectible, ...props }) => {
+  const { name, status, cost, lowestCost, nft } = collectible
+  const bnbBusdPrice = useBNBBusdPrice()
+
+  return (
+    <StyledCollectibleCard {...props}>
+      <CardBody p="8px">
+        <PreviewImage src={`/images/nfts/${nft.images.lg}`} height={320} width={320} mb="8px" />
```

Comment:
Tis is not going to work, we won't store nft images locally

Reply:
Yeah I figured as much. Will update it once it becomes clear how the images are available.